# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: widget/android/nsWindow.cpp
# Commit: 079ac6ad6133
# Full Hash: 079ac6ad6133db779d49b7347fe88b4c5adda09b
# Author: Hiroyuki Ikezoe <hikezoe.birchill@mozilla.com>
# Date: 2020-11-11 09:55:06
# Regressor Bug: 1674694
# File Overlap Count: 1
# Description:
#   Bug 1674694 - Use ConvertAPZHandledResult to convert mHandledByRootApzc for motion event cases. r=botond,geckoview-reviewers,agi
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D96483
# ==============================================================================

diff -r a2f0dae16c37 -r 079ac6ad6133 widget/android/nsWindow.cpp
--- a/widget/android/nsWindow.cpp	Wed Nov 11 00:33:28 2020 +0000
+++ b/widget/android/nsWindow.cpp	Wed Nov 11 00:33:30 2020 +0000
@@ -732,9 +732,7 @@
           break;
         case nsEventStatus_eConsumeDoDefault:
           returnResult->Complete(java::sdk::Integer::ValueOf(
-              (result.mHandledResult == Some(APZHandledResult::HandledByRoot))
-                  ? INPUT_RESULT_HANDLED
-                  : INPUT_RESULT_HANDLED_CONTENT));
+              ConvertAPZHandledResult(result.mHandledResult.value())));
           break;
         default:
           MOZ_ASSERT_UNREACHABLE("Unexpected nsEventStatus");
